Documentation/toc.yml

```diff
@@ -97,20 +99,32 @@
         href: Diagnostics/ConfiguringDiagnostics.md
       - name: Using the Visual Profiler
         href: Diagnostics/UsingVisualProfiler.md
-  - name: Other Features
+    - name: Extension Service Creation Wizard
+      href: Utilities/ExtensionServiceCreationWizard.md
+  - name: Experimental Features
     items:
+    - name: Spatial Understanding
+      href: TODO.md
     - name: QR Tracking
       href: TODO.md
     - name: Spatial Audio
       href: TODO.md
   - name: Advanced topics
     items:
-    - name: Performance
-      href: Performance/PerfGettingStarted.md
     - name: Shared Experiences
       href: TODO.md
-    - name: Moving your project from HTK to MRTK
-      href: HTKToMRTKPortingGuide.md
+- name: How-to Guides 
+  items:
+  - name: How to setup global voice commands 
+    href: TODO.md
+  - name: How to setup a scene to be used for both HoloLens and VR with proper camera and scene content height
```

Comment:
This is a bit long. Maybe just call it How to setup a scene for Mixed Reality?

Reply:
good idea this is long for a place holder.